### ISSUE TITLE
Support openmediavault 3.0.15

### ISSUE THREAD

**Issue by Flachzange** [OpenMediaVault-Plugin-Developers/openmediavault-jdownloader#3]
Please add package dependency for openmediavault 3.0.15:

```
The following packages have unmet dependencies:
 openmediavault-jdownloader : Depends: openmediavault (= 3.0.13) but 3.0.15 is to be installed
```

**Comment by davidh2k** [comment]
That will be done once the package is actually adapted to work with the 3.0.15. The database structure was changed in 3.0.15, so every plugin has to undergo certain changes to be adapted to 3.0.15. This is not done by simply changing a single line in the control file.

Greetings
David

**Comment by Flachzange** [comment]
Fair enough. I didn't know that they made these serious changes in 3.0.15. 

Looking forward to the adaptations :-)

Thanks for your effort.

**Comment by fightforlife** [comment]
When can we expect the new 3.2 Version on OMV-Extras?
Thanks in advance!

**Comment by ryecoaaron** [comment]
omv-extras 3.2 was released 4 days ago.

**Comment by fightforlife** [comment]
Sorry I meant the jdownloader plugin version 3.2 with the commit which should enable omv3 usage

**Comment by Flachzange** [comment]
Is there any update? Obviously there have been made some changes in the code to make the plugin work with 3.0.15. Is this still work in progress?

Thanks!

**Comment by cptjhmiller** [comment]
It is almost finished just need to track down an error that keeps popping up to do with the new datamodel.

**Comment by Flachzange** [comment]
:+1:  Thx

**Comment by samsal** [comment]
Any news on this?

**Comment by cptjhmiller** [comment]
Not yet. tbh I have not been able to look into the error as I have been converting some other plugins.
Once the one i am currently working on is finished I will fix this error.

Sorry for delay.

**Comment by samsal** [comment]
No problem. Thank you and keep up the good work.

**Comment by cptjhmiller** [comment]
Error has been fixed and a new version will be coming soon (i hope tonight), open a new issue if problem persists.
